renku_notebooks/api/classes/server.py

```diff
@@ -438,6 +431,12 @@ def get_annotations(self):
             f"{prefix}projectName": self.project,
             f"{prefix}requested-image": self.image,
             f"{prefix}repository": None,
+            f"{prefix}hibernation": None,
+            f"{prefix}hibernation-branch": None,
+            f"{prefix}hibernation-commit-sha": None,
+            f"{prefix}hibernation-dirty": None,
+            f"{prefix}hibernation-synchronized": None,
+            f"{prefix}hibernation-date": None,
```

Comment:
Should we just nest this and store the annotation as json? What do you think? I am going back and forth on the idea.

Reply:
So it would be something like:
```
f"{prefix}hibernation": '{"branch": "fdsfsd", "date": "some-date"}'
```

Then when we return this from the notebooks service in the API we use json.load and run it through marshmallow.

Reply:
UI requested to have them as separate fields since it's easier to deal with. We also have a field like that (`"renku.io/hibernation"`) that includes the json for hibernation values.